mailbox/tika/src/test/java/org/apache/james/mailbox/tika/TikaTextExtractorTest.java

```diff
@@ -89,15 +115,15 @@ public void pdfTest() throws Exception {
         InputStream inputStream = ClassLoader.getSystemResourceAsStream("documents/PDF.pdf");
         assertThat(inputStream).isNotNull();
         assertThat(textExtractor.extractContent(inputStream, "application/pdf", "PDF.pdf").getTextualContent())
-            .isEqualTo("\nThis is an awesome document on libroffice writter !\n\n\n");
+            .isEqualTo("This is an awesome document on libroffice writter !\n\n\n");
```

Comment:
it's where equality doesn't make sense, I'm sure there are other matchers that would better fit